src/content/docs/new-relic-solutions/tutorial-landing-page.mdx

```diff
@@ -3,15 +3,15 @@ title: Tutorials
 freshnessValidatedDate: 2023-09-28
 ---
 
-This page contains all of our tutorials. Whether you're just starting your New Relic journey, or you consider yourself an observability pro there are tutorials for everyone. Learn more about specific capabilities, push your observability to the next level, or simply learn about the power of the New Relic platform.
+This page contains all of our tutorials. Whether you're just starting your New Relic journey, or consider yourself an observability professional, there are tutorials for everyone. Learn more about specific capabilities, push your observability to the next level, or simply learn about the power of the New Relic platform.
```

Comment:
@techpubswriter the sentence starting out with "Whether..." originally had two independent clauses (subject + verb) on either side of the  comma and the conjunction "or." By dropping the "you" in the original, you'd also want to drop the comma. In this case, I don't recommend that since each clause is a bit lengthy. Let's keep the original "you" in this sentence.

As far as the tip to switch to "professional" from "pro," this could be helpful, but I think our docs are trying for a less formal tone. The shortened form of "pro" seems to capture that.